src/composables/trade/useGnosis.ts

```diff
@@ -173,16 +173,25 @@ export default function useGnosis({
         owner: account.value
       });
 
+      const sellAmount = exactIn.value
+        ? tokenInAmountInput.value
+        : formatUnits(quote.maximumInAmount, tokenIn.value.decimals).toString();
+
+      const buyAmount = exactIn.value
+        ? formatUnits(
+            quote.minimumOutAmount,
+            tokenOut.value.decimals
+          ).toString()
+        : tokenOutAmountInput.value;
```

Comment:
This will help the user understand where the amounts are coming from when exploring Gnosis explorer.